cmd/api-server/main.go

```diff
@@ -126,16 +124,32 @@ func main() {
 
 	inspector := commons.CreateImageInspector(cfg, configMapClient, secretClient)
 
+	// Connect to NATS
```

Comment:
Moving NATS first staggers the gRPC connection a bit when we start agent and control plane together. gRPC backoff is quite steep, but I want to avoid tweaking it until absolutely needed as these defaults are there for a  reason.